src/Symfony/Component/Workflow/Metadata/MetadataStoreInterface.php

```diff
@@ -33,7 +33,9 @@ public function getTransitionMetadata(Transition $transition): array;
      *
      * @param string|Transition|null $subject Use null to get workflow metadata
      *                                        Use a string (the place name) to get place metadata
-     *                                        Use a Transition instance to get transition metadata
+     *                                        Use a Transition instance to get tansition metadata
+     *
+     * @return string|null
```

Comment:
metadata values are not restricted to string or null AFAICT. This should be `mixed`